R/collectors.R

```diff
@@ -216,6 +217,9 @@ guess_parser <- function(x, locale = default_locale()) {
 #'
 #' # parse_factor generates a warning & problems
 #' x2 <- parse_factor(x, levels)
+#'
+#' # Using an argument of `NULL` will generate levels based on values of `x`
+#' x2 <- parse_factor(x, levels = NULL)
```

Comment:
And actually you are right, they are ordered by how they appear in the input, forgot how I implemented this.